addons/website/static/src/snippets/s_rating/001.scss

```diff
@@ -0,0 +1,21 @@
+
+.s_rating[data-vcss="001"] {
+    &.s_rating_2x .fa {
+        font-size: 2em;
+    }
+    &.s_rating_3x .fa {
+        font-size: 3em;
+    }
```

Comment:
In fact, is it not possible to just use a data-select-class="fa-2x" and fa-3x and apply it on icons with data-apply-to ? Seems like it would be less css without any more JS ?